### PR TITLE
docs: improve readme

### PR DESCRIPTION
**Pull request by bra-i-am**
## Description
This PR modifies the readme to have consigned the primary goal of this MFE and shows how to have it in a basic development environment.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,213 +1,69 @@
...
-
-   What is this MFE?  Add a 2-3 sentence description of what it is and what it
-   does.
-
-This is the Awesome MFE.  It was built to provide an unmatched learning
-experience, with improved tools for both randomized goodness and the ability to
-directly reference amaze-blocks in existing courses. This experience is powered
-by the new Fantastico storage engine.
+***************
+The Validation Panel is a micro-frontend (MFE) that empowers course authors to initiate validation processes for the courses they intend to publish on the platform. These processes are routed to validator bodies for a thorough course review, feedback, and determination of approval or disapproval. This MFE offers a user-friendly interface, streamlining the course validation process for both authors and validators.
```
Comment (dcoa) [inline README.rst]:
```suggestion
The Validation Panel is a micro-frontend (MFE) that enables course authors to initiate validation processes for the courses they intend to publish on the platform. These processes are routed to validators for a thorough review, feedback, and determination of approval or disapproval. This MFE offers a user-friendly interface, streamlining the course validation process for both authors and validators.

This application was designed for UNESCO's Open edX instance and depends on the Platform Global Teacher Campus Plugin*
```

\* Should be linked to the backend https://github.com/eduNEXT/platform-global-teacher-campus/tree/main#platform-global-teacher-campus-plugin

**Comment by dcoa** [comment]
Thanks for the PR @bra-i-am 

There are some points that I think should improve:

1. You need to be clear about the configuration of the platform, I mean, is a must to define `VALIDATION_PANEL_MFE_URL` in the settings. 

2. I would rather have developing instructions like [example](https://github.com/openedx/frontend-app-ora-grading#developing) because we use Tutor but should not be limited to that, in addition, making reference to course-authoring without an explanation will be confusing. 

3. Don't forget the deploying instructions (here could be the right point to add the tutor recommendation) and use the tutor instructions to add a new MFE.

I added @MaferMazu as a reviewer understanding her wide knowledge of technical documentation.

File: README.rst (hunk 1 above)
Comment (MaferMazu) [reply README.rst]: I agree with this suggestion.

**Comment by MaferMazu** [comment]
I will add that it would be cool to have a no-developer installation. For example: https://github.com/eduNEXT/platform-global-teacher-campus#getting-started.
Something like:
- Add X in your config.yml
- Save with `tutor config save`
- Create the mfe image `tutor images build mfe`
- Launch the platform `tutor local launch`

I think it's important because we probably will install this plugin more than we want to continue to develop over this, and this is not installed by default by tutor-mfe like other mfes.

Btw I only suggest; you feel free to improve my example xD

Another btw, this PR should be a "docs" instead of "refactor".

File: README.rst [hunk 2]
```diff
@@ -5,65 +5,89 @@ frontend-app-validation-panel
...
+
+Deployment
+============
+This component follows the standard deploy process for MFEs. For details, see
+the `MFE production deployment guide`_
+
+Now, if you prefer to use Tutor, you will need to create an image containing the new MFE. With this in mind, you have to execute the following steps:
 
-#. Create a ``docker-compose.override.yml`` file in the ``env > dev`` directory from the root of your new Tutor project
+1. Declare the new MFE adding the following code in your config.yml
```
Comment (dcoa) [inline README.rst]: take in mind https://discuss.openedx.org/t/config-tutor-mfe/10152/7?u=dcoa and https://github.com/overhangio/tutor-mfe/tree/v16.1.1#adding-new-mfes

Comment (dcoa) [reply README.rst]: Just this small change and I think the PR is ready for merge

Comment (bra-i-am) [reply README.rst]: I already added not only how to declare the MFE with the command line, but also a link to the tutor-mfe documentation where the process is better explained. I'm pending if it's necessary any other change

Thank you for your feedback c:

**Review by dcoa — APPROVED** [review]

**Pull request merged**